zulip/tests/test_default_arguments.py

```diff
@@ -16,8 +16,11 @@
 class TestDefaultArguments(TestCase):
 
     def test_invalid_arguments(self):
+        # type: () -> None
         parser = zulip.add_default_arguments(argparse.ArgumentParser(usage="lorem ipsum"))
-        with self.assertRaises(SystemExit) as cm, patch('sys.stderr', new=six.StringIO()) as mock_stderr:
+        cm = self.assertRaises(SystemExit)  # type: ignore # error: No variant of "assertRaises" matches argument types
+        mock_stderr = patch('sys.stderr', new=six.StringIO())
+        with cm as cm, mock_stderr as mock_stderr:
```

Comment:
@derAnfaenger Not really, it would make the test fail:
```
ERROR: test_invalid_arguments (tests.test_default_arguments.TestDefaultArguments)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/python-zulip-api/zulip/tests/test_default_arguments.py", line 27, in test_invalid_arguments
    self.assertTrue(mock_stderr.getvalue().startswith("""usage: lorem ipsum
AttributeError: '_patch' object has no attribute 'getvalue'
```

Reply:
@neiljp Could you explain again how this line could be improved if `typing` is fixed? The line you suggested looks exactly like the original now :)